src/mobile/collections/fairs.coffee

```diff
@@ -12,15 +12,6 @@ module.exports = class Fairs extends Backbone.Collection
 
   model: Fair
 
-  comparator: (fair) ->
-    sizes =
-      'x-large' : 1
-      'large' : 2
-      'medium' : 3
-      'small' : 4
-      'x-small' : 5
-    sizes[fair.get('banner_size')]
```

Comment:
I did a brief search in Force under mobile and looked like there is no other places do something like `fairs.fetch`. Modifying the default comparator here might be fine, although this sorting order makes some sense to me (sorting by relative size.) A safer alternative to keep the default sorting for other use cases is to [override](http://backbonejs.org/#Collection-comparator) it just for the fairs we fetch from the fair organizer app. We can do something like
```coffeescript
fair = new Fairs
fair.comparator = #...
```